### ISSUE TITLE
ZEN V12 - Captive portal displaying latin characters incorrectly as ASCII <only on central DIV>

### ISSUE THREAD

**Issue by grazzianos** [inverse-inc/packetfence#7281]
I'm testing the ZEN v12 image, manually migrating some configurations for homologation. When I tested captive portal, a strange behavior occourred: the central DIV was displaying latin chatacters incorrectly, but in other parts of the page, the characters was displayed correctly.

**To Reproduce**
1. Go to 'Configuration' > ' Advanced Access Configuration' > Portal Modules;
2. Create a new root module;
3. Create a new 'Message' module;
4. Insert this message in the right field: 

```
<h3>Você não tem permissão para acessar esta rede local!</h3>
<br>
Para obter acesso, clique em continuar e entre com as credenciais autorizadas para autenticação. 
<br><br>
<p><i>Ao validar este método, os registros de sua navegação serão armazenados, podendo ser utilizados como material probatório para eventuais sanções, de acordo com nossa Política de Usabilidade.</i></p>
```

6. Go to 'Policies and Access Control' > 'Connection Profiles';
7. Edit 'default' profile, setting as 'Root Portal Module' the module created on step 2;
8. Click on preview;
9. See the error.

**Expected behavior**
See the message module correctly displaying Latin characters. Apparently, the system changed the code format of the inserted text to ASCII.

**Desktop:**
 - OS: Windows 10;
 - Browser: Chrome;
 - Version 106

**Additional context**
I saw a similar closed issue (https://github.com/inverse-inc/packetfence/issues/3952), but in my case, occurred only on central DIV of the captive page. Someone with the same issue? Can someone help me?

I need to implement v12 soon in my office.

**Screenshots**
Bellow, the images of my issue:

![Captura de tela 2022-10-16 200216](https://user-images.githubusercontent.com/15256997/196295895-050f1244-07c5-4dbe-9ca7-b8b4c1cc4287.jpg)

![Captura de tela 2022-10-16 200233](https://user-images.githubusercontent.com/15256997/196295911-b92369e6-6706-4682-a623-6c89c87d366a.jpg)

**Comment by nqb** [comment]
Hello @grazzianos,

You didn't use your template to open issues. Please update your issue description. We need steps in order to replicate issue.

**Comment by grazzianos** [comment]
Hello @nqb.

Sorry, when I clicked on new issue, the template didn't appear to me. Issue properly updated.

**Comment by nqb** [comment]
Hello @grazzianos,

Thanks ! I'm able to replicate issue.

Issues I noticed:
- Description of a source which contains UTF8 characters is not displayed correctly on captive portal
- Description of a portal module which contains UTF8 characters is not displayed correctly on captive portal
- Display a message which contains UTF8 characters using Message module doesn't work

Important: If I add UTF8 characters inside aup_text.html, they appear correctly.

Conf files to replicate:

```
#profiles.conf
[..]
[french_sms_profile_source_on_portal]
advanced_filter=
locale=
filter=connection_type:Inline
sources=french_sms

[french_sms_profile_with_portal_module]
filter=connection_type:Inline
root_module=french_sms_root_module
locale=
advanced_filter=

#portal_modules.conf
[..]
[french_sms_root_module]
description=french_sms_root_module
modules=choice
type=Root

[french_sms_auth_portal_module]
pid_field=telephone
signup_template=signin.html
with_aup=1
fields_to_save=
actions=
aup_template=aup_text.html
description=french_sms_auth_portal_module_description
type=Authentication::SMS
custom_fields=
source_id=french_sms

[null_authentication_module]
custom_fields=
source_id=null
type=Authentication::Null
description=null_authentication_module_desc
aup_template=aup_text.html
actions=
fields_to_save=
with_aup=1
signup_template=signin.html

[choice]
modules=french_sms_auth_portal_module,null_authentication_module,french_sms_auth_portal_module_copy
show_first_module_on_default=disabled
template=content-with-choice.html
type=Choice
description=choice_desc
actions=

[french_sms_auth_portal_module_copy]
description=France Télé auth module description
aup_template=aup_text.html
actions=
source_id=french_sms
custom_fields=
type=Authentication::SMS
signup_template=signin.html
pid_field=telephone
fields_to_save=
with_aup=1

#authentication.conf
[..]
[french_sms]
set_access_durations_action=
local_account_logins=0
type=SMS
description=France Télé SMS source description
pin_code_length=6
message=PIN: $pin
password_length=8
hash_passwords=bcrypt
sms_carriers=
sms_activation_timeout=10m
local_account_expiration=0s
create_local_account=no

[null]
description=Null Source
type=Null
email_required=no
dynamic_routing_module=AuthModule

[null rule catchall]
description=catchall
class=authentication
match=all
action0=set_role=guest
action1=set_access_duration=1D
status=enabled
```




@jrouzierinverse, I think these issues are related to #7106. Could you look into this ?

**Comment by grazzianos** [comment]
Hello @nqb,

Thanks for the quickly appreciation.

**Comment by nqb** [comment]
Fix has been backported to maintenance/12.0
